lib/config.js

```diff
@@ -117,7 +119,13 @@ Config.prototype._parseFile = function(target, filePath) {
     if (target === 'database') {
       this.notes.databaseRules = 'json';
     } else if (target === 'database.rules') {
-      return fs.readFileSync(fullPath, 'utf8');
+      var rules = fs.readFileSync(fullPath, 'utf8');
+      if (validateJsonRules(rules)) {
+        return rules;
+      }
+      utils.logWarning(chalk.bold.yellow('database: ') + chalk.bold(fullPath) + ' must have an outer ' + chalk.bold('rules') + ' key, for example:');
+      logger.info('{\n\t"rules": {".read": false, ".write": false}\n}');
```

Comment:
Please add an extra newline in front here, and do `logger.warn` so it's the same severity as the above notice.